osh/client/commands/cmd_find_tasks.py

```diff
@@ -71,6 +93,8 @@ def run(self, *args, **kwargs):
         else:
             # nvr is default one, so we don't care if it's specified
             query['nvr'] = query_string
+        if state_name is not None:
+            query['state'] = get_state_from_name(state_name)
```

Comment:
No, there is no need to update that line of code because those two expressions in https://github.com/openscanhub/openscanhub/pull/183#discussion_r1394148929 are equivalent.  The `__get__` magic method of kobo's `Enum` class returns the ID of the state that is stored in the database.  Moreover, it is also more efficient with respect to the size of the `TASK_STATES` enum.